### ISSUE TITLE
`Window::update` blocks forever

### ISSUE THREAD

**Issue by neachdainn** [emoon/rust_minifb#134]
```bash
$ uname -a
Darwin hircine 17.6.0 Darwin Kernel Version 17.6.0: Tue May  8 15:22:16 PDT 2018; root:xnu-4570.61.1~1/RELEASE_X86_64 x86_64 i386 MacBookPro11,4 Darwin

$ rustc --version
rustc 1.40.0 (73528e339 2019-12-16)
```

Using minifb v0.15, calling `Window::update` in a loop causes the update function to block forever. This problem does not seem to happen when using minifb v0.14. Below is a minimal example that demonstrates the bug on my machine. The program will block on line 10 after only a few iterations of the loop.

```rust
use minifb::Window;
use std::time::{Duration, Instant};

fn main() {
    let mut window = Window::new("Test", 800, 600, Default::default()).unwrap();
    let start = Instant::now();
    let run_time = Duration::from_secs(2);

    while window.is_open() {
        window.update();

        let dur = Instant::now().duration_since(start);
        if dur >= run_time {
            break;
        }
    }
}
```

**Comment by emoon** [comment]
Thanks for the report. I will take a look

**Comment by emoon** [comment]
Fixed in 0.15.2 Thanks again for the repot!
